apps/passport-server/scripts/scratch.ts

```diff
@@ -1,44 +1,154 @@
+import { insertDevconnectPretixTicket } from "../src/database/queries/devconnect_pretix_tickets/insertDevconnectPretixTicket";
+import { insertPretixEventsInfo } from "../src/database/queries/pretixEventInfo";
+import { insertPretixItemsInfo } from "../src/database/queries/pretixItemInfo";
+import {
+  insertPretixEventConfig,
+  insertPretixOrganizerConfig
+} from "../src/database/queries/pretix_config/insertConfiguration";
 import { logger } from "../src/util/logger";
 
-dotenv.config({ path: path.join(__dirname, ".env") });
-
-const orgUrl: string = process.env.ORG_URL!;
-const token: string = process.env.PRETIX_TOKEN!;
-
-if (!orgUrl || !token) {
-  logger(`missing orgUrl or pretix token`);
-  process.exit(0);
-}
-
-async function scratch(): Promise<void> {
-  const api = new DevconnectPretixAPI();
-  const events = await api.fetchAllEvents(orgUrl, token);
-
-  logger();
-  logger(`EVENTS: ${orgUrl}`);
-  for (const e of events) {
-    const items = await api.fetchItems(orgUrl, token, e.slug);
-    logger(`EVENT name: '${e.name.en}'; slug: '${e.slug}'`);
-    items.forEach((i) => {
-      logger(`  ITEM id: '${i.id}'; name: '${i.name.en}'`);
-    });
-    const itemsStr = `  ITEMS: {${items.map((i) => i.id).join(", ")}}`;
-    logger(itemsStr);
-  }
-
-  logger();
-}
-
-scratch()
-  .then(() => {
-    logger("DONE");
-    process.exit(0);
-  })
-  .catch((e) => {
-    logger(e);
-    process.exit(0);
-  });
+dotenv.config({ path: path.join(__dirname, "..", ".env") });
+
+yargs
+  .scriptName("yarn scratch")
+  .usage("$0 <cmd> [args]")
+  .command(
+    "fetch",
+    "Fetch all events and items from pretix",
+    (yargs) => {},
+    async function (argv) {
+      const orgUrl: string = process.env.ORG_URL!;
+      const token: string = process.env.PRETIX_TOKEN!;
+
+      if (!orgUrl || !token) {
+        throw new Error(`missing orgUrl or pretix token`);
+      }
+
+      const api = new DevconnectPretixAPI();
+      const events = await api.fetchAllEvents(orgUrl, token);
+
+      logger();
+      logger(`EVENTS: ${orgUrl}`);
+      for (const e of events) {
+        const items = await api.fetchItems(orgUrl, token, e.slug);
+        logger(`EVENT name: '${e.name.en}'; slug: '${e.slug}'`);
+        items.forEach((i) => {
+          logger(`  ITEM id: '${i.id}'; name: '${i.name.en}'`);
+        });
+        const itemsStr = `  ITEMS: {${items.map((i) => i.id).join(", ")}}`;
+        logger(itemsStr);
+      }
+
+      logger();
+    }
+  )
+  .command(
+    "new-dev-event [name]",
+    "Create a new event for development",
+    (yargs) =>
+      yargs.positional("name", {
+        type: "string",
+        default: "Zero Nexus: Exploring the Frontiers of Zero-Knowledge",
+        describe: "the name of the event"
+      }),
+    async function (argv) {
+      const db = await getDB();
+      const organizerConfigId = await insertPretixOrganizerConfig(
+        db,
+        "http://localhost",
```

Comment:
When the sync service is enabled, it's going to be reporting errors when it tries to fetch this URL. It's also going to cause an HTTP request to whatever is running on `localhost:80`, which might not be what you want.

Reply:
This is just a dummy value for test purposes - @robknight do you have a better suggestion for what to put here?

Reply:
I still don't really understand why fetching dummy data from the development pretix environment wasn't an option, so I can only really suggest that.

Reply:
If I'm following, do you mean fetching from a dev pretix event itself instead of manually inserting into the db?

Reply:
> If I'm following, do you mean fetching from a dev pretix event itself instead of manually inserting into the db? Because if we decide to go down that route, we can get rid of all of the manual DB writes etc... cc @ichub @rrrliu

that would be a great option - @rrrliu can help set that up for you guys.

Reply:
We can remove this now that you guys have sync service set up!